projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -312,6 +312,10 @@ export class TuiInputDateRangeComponent
         this.toggle();
         this.focusInput();
 
+        if (!range) {
```

Comment:
`range` can be only `TuiDayRange`. So, this condition is always false according to typing

Reply:
I mean you might need to fix the type or smth :)

Reply:
thanks! fixed